internal/cmd/timestamp_flag.go

```diff
@@ -0,0 +1,9 @@
+package cmd
+
+import "github.com/spf13/cobra"
+
+var timestampFlag string
+
+func addDbTimestampFlag(cmd *cobra.Command) {
+	restoreCmd.Flags().StringVarP(&timestampFlag, "timestamp", "t", "", "UTC timestamp for the version of database at a given point in time.")
```

Comment:
What's the format of the timestamp? Just int? In what unit?

Reply:
Ok. I can see it here https://github.com/tursodatabase/turso-cli/pull/637/files#diff-e963bae42bbc05e1ecbaa07a34c3a585f0278493311e24f2a4ab030e8267c940R19